server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -64,6 +64,19 @@ def deploy
       end
     end
 
+    def redeploy
+      begin
```

Comment:
Is there a reason we went with a new API instead of adding a param to the deploy? i.e. ```/deploy/id=8abadef&redeploy=true```?